recipes/pass/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.3.1" %}
+{% set version_tar = "0-3-1" %}
+{% set name = "pass" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bcgsc/{{ name | upper }}/releases/download/v{{ version }}/pass_v{{ version_tar}}.tar.gz
+  sha256: 813fab567b1159c6dc32ed8ad537d19c50e70bde347b4e8b1249a4ece1de4383
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pass', max_pin="x") }}
+
+requirements:
+  run:
+    - perl
+
+test:
+  commands:
+    - PASS -h | grep "Usage"
+
+about:
+  summary: Proteome Assembler with Short Sequence peptides
+  home: https://github.com/bcgsc/PASS
+  license: GPL-3.0
```

Comment:
can you please use a SPDX identifier here?

Reply:
@bgruening - For sure! Updated the license